src/runtime_src/core/common/drv/xrt_cu.c

```diff
@@ -266,6 +266,8 @@ static inline void process_cq(struct xrt_cu *xcu)
 		xcmd = list_first_entry(&xcu->cq, struct kds_command, list);
 		set_xcmd_timestamp(xcmd, xcmd->status);
 		xrt_cu_circ_produce(xcu, CU_LOG_STAGE_CQ, (uintptr_t)xcmd);
+		if (unlikely(xcmd->status == KDS_SKCRASHED))
+		  xcu->bad_state = true;
```

Comment:
For a one line `if` we do not need likely/unlikely. The following will suffice:
`xcu->bad_state = (xcmd->status == KDS_SKCRASHED);`